### ISSUE TITLE
Loss Functions

### ISSUE THREAD

**Issue by Jkoles28** [Jkoles28/CS373#12]
### Title
Loss Functions and Their Use In Neural Networks
### URL
https://towardsdatascience.com/loss-functions-and-their-use-in-neural-networks-a470e703f1e9
### Summary 
Loss functions are used to measure the performance of a neural network and are used in backpropagation to update the model’s hyperparameters. Different types of loss functions include mean squared error (MSE), cross-entropy, and hinge loss. Custom loss functions can be implemented for image and video data processing.
### Key Points 
A loss function is a function that compares the target and predicted output values; measures how well the neural network models the training data. When training, we aim to minimize this loss between the predicted and target outputs.
The hyperparameters are adjusted to minimize the average loss — we find the weights, wT, and biases, b, that minimize the value of J (average loss).
![image](https://user-images.githubusercontent.com/81580102/221004796-416b528c-9aba-4f26-9d4a-eb9a3fbf10ad.png)

**Comment by Jkoles28** [comment]
![image](https://user-images.githubusercontent.com/81580102/225148689-f81e738c-6f26-4212-9b5d-abeced5a296a.png)
